lib/msal-common/src/client/AuthorizationCodeClient.ts

```diff
@@ -557,9 +557,11 @@ export class AuthorizationCodeClient extends BaseClient {
                 parameterBuilder.addSid(request.sid);
             } else if (request.account) {
                 const accountSid = this.extractAccountSid(request.account);
-                const accountLoginHintClaim = this.extractLoginHint(
-                    request.account
-                );
+                // Skip opaque login_hint claim if domain hint is present
+                const accountLoginHintClaim = request.domainHint
```

Comment:
Can we log a warning when we do this? Opaque loginHint should be most preferred so we should indicate to developers they should stop passing domainHint

Reply:
Good point, thanks! Added a warning message.